modules/campaignion_email_protest/campaignion_email_protest.install

```diff
@@ -1,21 +1,13 @@
 <?php
 
 /**
- * @file
- * campaignion_email_protest.install
+ * Enable the newly separated content-type module.
  */
-
-/**
- * Implements hook_enable().
- */
-function campaignion_email_protest_enable() {
-  variable_set('webform_node_email_protest', 1);
-
-  $webform_settings = variable_get('webform_template_src', array());
-  $webform_settings['email_protest'] = 'email_protest';
-  variable_set('webform_template_src', $webform_settings);
-
-  $webform_settings = variable_get('webform_template_dest', array());
-  $webform_settings['email_protest'] = 'email_protest';
-  variable_set('webform_template_dest', $webform_settings);
+function campaignion_email_protest_update_1() {
```

Comment:
Does the update have any implications for existing overrides of the campaignion_email_protest_templates feature?

Reply:
Nice catch. I think `features_modules_enabled()` would have reverted the feature on it’s installation. I’ve added a hack to avoid that.